Assets/Scripts/Interactables/Dialogue/Conversation.cs

```diff
@@ -16,4 +16,6 @@ public class Conversation : ScriptableObject
 {
     public string npcVoice;
     public Line[] lines;
+    public bool isAutoplayed = false;
+    public bool shouldFollow = false;

```

Comment:
I think we should have the settings above `lines`? Because if `lines` is expanded, it'll hide all the settings below it.